src/developer/web-api/tracker.md

```diff
@@ -2639,11 +2641,13 @@ although it might have multiple rows for each data element value.
 #### Event data value change logs { #webapi_event_data_value_change_logs }
 `GET /api/tracker/events/{uid}/changeLogs`
 
-This endpoint retrieves change logs for the data values of a specific event. It returns a list of all event data values that have changed over time for that particular event.
+This endpoint retrieves change logs for the data values of a specific event. It returns a list of all event data values and event properties (`occurredAt`, `scheduledAt`, and `geometry`) that have changed over time for the specified event.
 
 |Parameter|Type|Allowed values|
 |---|---|---|
 |path `/{uid}`|`String`|Event `UID`.|
+|`order`|`String`|Name and sort direction pair in the format `propName:sortDirection`.<br><br>Example: `createdAt:desc`<br><br>`propName` is case-sensitive. Valid sortDirection values are `asc` and `desc`. `sortDirection` is case-insensitive and defaults to `desc`. Supported properties are `createdAt`, `dataElement`, `property` and `username`, only one at a time.|
```

Comment:
Users can order by `property`? Not sure if it's clear what that is. Is that explained somewhere?

Will that also be supported in
`/api/tracker/trackedEntities/{uid}/changeLogs`

Reply:
That's explained here: https://github.com/dhis2/dhis2-docs/pull/1462/files#diff-4b5dabc4706dd8ae8c4c71cc43b6c8d08fa520c861a63cae3908ac407f201fffR2644

No, only event change logs support `property`